.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+** Checklist **
+- [ ] This PR includes no more than 5 new icons
+- [ ] All icons include `width="28" height="24"` in their attributes
+- [ ] All icons have solid white backgrounds
+- [ ] All icons have a corresponding entry in `db/payment_icons.yml`
+- [ ] I am confident that all icons are clear and easy to read/understand
```

Comment:
This one was the biggest inspiration for me to add this, let's keep it and see how useful it is